students/GKim/lesson02/grid_printer.py

```diff
@@ -0,0 +1,118 @@
+
+"""
+Shortened version
+"""
+
+def print_symbol(c, a =2,s1="",s2=""): 
```

Comment:
be consistent with your spacing, ideally following PEP8:

```
def print_symbol(c, a=2, s1="", s2=""):
```
make it a little more readable ...